projects/demo/src/modules/components/input-tag/examples/8/index.html

```diff
@@ -0,0 +1,18 @@
+<tui-input-tag
+    class="input"
+    [tuiTextfieldLabelOutside]="true"
+    [formControl]="control"
+    [rows]="3"
```

Comment:
Is this scroll expected ?

(video shows what happens when i type value and press "enter")

https://user-images.githubusercontent.com/35179038/210048874-1a6c9673-bb92-47d5-b386-8ccd30cf06c1.mov